test/e2e/operator.go

```diff
@@ -272,3 +277,101 @@ var _ = Describe("ARO Operator - Conditions", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 })
+
+var _ = Describe("ARO Operator - Azure NSG Reconciler", func() {
+	var location, resourceGroup, subnet, vnet string
+	var correctNSG *string
+	var dummynsg mgmtnetwork.SecurityGroup
+	type subnetDescriptor struct {
+		resourceGroup string
+		vnetName      string
+		subnetName    string
+		location      string
+	}
+	enableReconcileNSG := func() {
+		instance, err := clients.AROClusters.AroV1alpha1().Clusters().Get(context.Background(), arov1alpha1.SingletonClusterName, metav1.GetOptions{})
```

Comment:
Someone on the team gave me this feedback on an e2e (maybe @troy0820), so thought I'd pass it along.

Rather than use `context.Background()` in each Get/List/Update, you can define it in a `ctx := context.Background() ` var and use it like you would outside of a test:
```suggestion
		instance, err := clients.AROClusters.AroV1alpha1().Clusters().Get(ctx, arov1alpha1.SingletonClusterName, metav1.GetOptions{})
```
It's functionally the same since it's just an empty context, but avoids having to define a new empty context on each Get/List/Update.